.gitignore

```diff
@@ -14,4 +14,6 @@
 
 ### Go Patch ###
 /vendor/
-/Godeps/
\ No newline at end of file
+/Godeps/
+
+.idea
```

Comment:
IDE ignores should be done in your global gitignore not per project. Please also ensure trailing line return.